apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -15,8 +15,29 @@ var log = logf.Log.WithName("baremetalhost-validation")
 func (host *BareMetalHost) validateHost() []error {
 	log.Info("validate create", "name", host.Name)
 	var errs []error
+	var bmcAccess bmc.AccessDetails
 
-	if err := validateRAID(host.Spec.RAID); err != nil {
+	if host.Spec.BMC.Address != "" {
+		var err error
```

Comment:
We can remove this and use below line as;
`bmcAccess, err := bmc.NewAccessDetails(host.Spec.BMC.Address, host.Spec.BMC.DisableCertificateVerification)`

Reply:
We can't do this. Because the value of `bmcAccess` cannot be used outside of the `if`.